lib/redmine_webhook/webhook_listener.rb

```diff
@@ -26,10 +27,32 @@ def controller_issues_edit_after_save(context = {})
       issue = context[:issue]
       project = issue.project
       webhooks = Webhook.where(:project_id => project.project.id)
+      webhooks = Webhook.where(:project_id => 0) unless webhooks && webhooks.length > 0
       return unless webhooks
       post(webhooks, journal_to_json(issue, journal, controller))
     end
 
+    def controller_issues_bulk_edit_after_save(context = {})
+      return if skip_webhooks(context)
+      journal = context[:journal]
+      controller = context[:controller]
+      issue = context[:issue]
+      project = issue.project
+      webhooks = Webhook.where(:project_id => project.project.id)
+      webhooks = Webhook.where(:project_id => 0) unless webhooks && webhooks.length > 0
+      return unless webhooks
+      post(webhooks, journal_to_json(issue, journal, controller))
+    end
+
+    def model_changeset_scan_commit_for_issue_ids_pre_issue_update(context = {})
```

Comment:
I'm wondering if I can put it in this plugin.
Because this is a model hook, skip_webhooks cannot be called.

Reply:
For the time being, I decided to merge this change too.
If someone will say they want to skip, I will turn it off.